demo/site/src/common/components/BasicSwiper.tsx

```diff
@@ -0,0 +1,42 @@
+import "swiper/css";
+import "swiper/css/a11y";
+import "swiper/css/keyboard";
+import "swiper/css/navigation";
+import "swiper/css/pagination";
+
+import { useIntl } from "react-intl";
+import { A11y, Keyboard, Navigation } from "swiper/modules";
+import { Swiper, type SwiperProps } from "swiper/react";
+
+export const BasicSwiper = ({ children, modules = [], ...restProps }: SwiperProps) => {
+    const intl = useIntl();
+    return (
+        <Swiper
+            a11y={{
+                firstSlideMessage: intl.formatMessage({
+                    defaultMessage: "This is the first slide",
+                    id: "swiper.prevButton.firstSlide.ariaLabel",
+                }),
+                lastSlideMessage: intl.formatMessage({
+                    defaultMessage: "This is the last slide",
+                    id: "swiper.nextButton.lastSlide.ariaLabel",
+                }),
+                nextSlideMessage: intl.formatMessage({ defaultMessage: "Next slide", id: "swiper.nextButton.ariaLabel" }),
+                paginationBulletMessage: intl.formatMessage(
+                    {
+                        defaultMessage: "Go to slide {index}",
+                        id: "swiper.paginationBullet.ariaLabel",
+                    },
+                    { index: "{{index}}" }, // Swiper replaces {{index}} with the actual index
```

Comment:
Why do we need a variable here? Is it because react-intl tries to parse "{{index}}" in the message?

Reply:
and yes, react-intl parse {index} and expects a variable then

Reply:
I understand that we need a variable for swiper, but I'm not sure if we need one for react-intl. Wouldn't `Go to slide {{index}}` work as well?

Reply:
> react-intl parse {index} and expects a variable then

and throws an error, when no variable is given.